src/scandeval/utils.py

```diff
@@ -21,6 +21,21 @@
 from .exceptions import InvalidBenchmark
 
 
+def create_model_cache_dir(cache_dir: str, model_id: str) -> str:
+    """Create cache directory for a model.
+
+    Args:
+        cache_dir: The cache directory.
+        model_id: The model ID.
```

Comment:
For the sake of consistency.

```suggestion
        cache_dir: 
            The cache directory.
        model_id: 
            The model ID.
```